<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14182" started-at="2019-10-03T15:01:34Z" finished-at="2019-10-03T15:01:48Z">
    <groups>
      <group name="Regression">
        <method signature="setup_defineSetupTest.SET012()[pri:0, instance:com.vrt.testcases.setup_defineSetupTest@6fc6f14e]" name="SET012" class="com.vrt.testcases.setup_defineSetupTest"/>
      </group> <!-- Regression -->
    </groups>
    <test name="Default test" duration-ms="14182" started-at="2019-10-03T15:01:34Z" finished-at="2019-10-03T15:01:48Z">
      <class name="com.vrt.testcases.setup_defineSetupTest">
        <test-method status="PASS" signature="AssetCreationSetup()[pri:0, instance:com.vrt.testcases.setup_defineSetupTest@6fc6f14e]" name="AssetCreationSetup" is-config="true" duration-ms="94" started-at="2019-10-03T15:01:34Z" finished-at="2019-10-03T15:01:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AssetCreationSetup -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.vrt.testcases.setup_defineSetupTest@6fc6f14e]" name="Setup" is-config="true" duration-ms="11967" started-at="2019-10-03T15:01:34Z" finished-at="2019-10-03T15:01:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="SET012()[pri:0, instance:com.vrt.testcases.setup_defineSetupTest@6fc6f14e]" name="SET012" duration-ms="1793" started-at="2019-10-03T15:01:46Z" description="SET 012- UI_Verify if  _Number of Sensors _ (mandatory field) is displayed in the _Define Setup_ screen." finished-at="2019-10-03T15:01:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
FAIL: SET 012-Sensor Data field mandatory alert message not displayed or Wrong Alert msg expected [Setup Name is mandatory, please enter Setup Name] but found [Number of Sensors is mandatory, please enter Number of Sensors]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
FAIL: SET 012-Sensor Data field mandatory alert message not displayed or Wrong Alert msg expected [Setup Name is mandatory, please enter Setup Name] but found [Number of Sensors is mandatory, please enter Number of Sensors]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.vrt.testcases.setup_defineSetupTest.SET012(setup_defineSetupTest.java:275)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SET012 -->
        <test-method status="PASS" signature="Teardown(org.testng.ITestResult)[pri:0, instance:com.vrt.testcases.setup_defineSetupTest@6fc6f14e]" name="Teardown" is-config="true" duration-ms="94" started-at="2019-10-03T15:01:47Z" finished-at="2019-10-03T15:01:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=SET012 status=FAILURE method=setup_defineSetupTest.SET012()[pri:0, instance:com.vrt.testcases.setup_defineSetupTest@6fc6f14e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="PASS" signature="endReport_releaseMomory()[pri:0, instance:com.vrt.testcases.setup_defineSetupTest@6fc6f14e]" name="endReport_releaseMomory" is-config="true" duration-ms="234" started-at="2019-10-03T15:01:48Z" finished-at="2019-10-03T15:01:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport_releaseMomory -->
      </class> <!-- com.vrt.testcases.setup_defineSetupTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
